src/main/java/appeng/bootstrap/IBootstrapComponent.java

```diff
@@ -19,28 +19,9 @@
 package appeng.bootstrap;
 
 
-import net.minecraftforge.fml.relauncher.Side;
-
-
 /**
  * Bootstrap components can be registered to take part in the various initialization phases of Forge.
```

Comment:
This should probable state something like `See the individual subclasses for a specific forge initalization event`.